### PR TITLE
Fix HeliReturnToBase not exposing an host actor parameter

### LINKED ISSUE #12114: Helicopter ReturnToBase lua command ignores the given airfield.

### PR DESCRIPTION
**Pull request by abcdefg30**
Fixes #12114.

### CODE REVIEW

File: OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void ReturnToBase(Actor airfield = null)
 			if (isPlane)
 				Self.QueueActivity(new ReturnToBase(Self, false, airfield));
 			else
-				Self.QueueActivity(new HeliReturnToBase(Self, false));
+				Self.QueueActivity(new HeliReturnToBase(Self, false, airfield));
```
Comment (cjshmyr) [inline OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs]: Minor: Is it safe to fix the `Desc` tag and parameter to not use `airfield` but something generic like `destination`?

Comment (abcdefg30) [inline OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs]: Yes, updated.

File: OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs [hunk 2]
```diff
@@ -38,13 +38,13 @@ public void Move(CPos cell)
 		}
 
 		[ScriptActorPropertyActivity]
-		[Desc("Return to the base, which is either the airfield given, or an auto-selected one otherwise.")]
-		public void ReturnToBase(Actor airfield = null)
+		[Desc("Return to the base, which is either the destionation given, or an auto-selected one otherwise.")]
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Scripting/Properties/AircraftProperties.cs]: Oops, fixed.

**Review by pchote — APPROVED** [review]
I haven't written a script to test this, but the changes are consistent (for better or worse) with how planes work.

I'd like to think that the second approval can come from someone who can whip up a quick test script to make sure that this indeed works as advertised.

**Comment by reaperrr** [comment]
Code looks good to me and I didn't notice any in-game regressions, but I agree that we need confirmation from someone with a lua test script that this works as intended.

**Comment by abcdefg30** [comment]
For easier review:
```lua
	local pad = Actor.Create("hpad", true, { Owner = player, Location = CPos.New(30, 30) })
	local pad2 = Actor.Create("hpad", true, { Owner = player, Location = CPos.New(45, 30) })
	local hind = Actor.Create("hind", true, { Owner = player, Location = CPos.New(60, 30) })

	-- It normally returns to "pad", when no parameter is given
	hind.ReturnToBase(pad2)
```

**Review by reaperrr — APPROVED** [review]
Tested myself using the example code, works fine.

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/cc49e8f285e03bc001b20b8ed232326fd4957748...f2c1b6ac57c81c63d2f9229580b54c3c41695107)

**Pull request merged**